kratos/geometries/geometry.h

```diff
@@ -1279,6 +1279,20 @@ class Geometry
         return result;
     }
 
+    /**
+     * @brief It returns a vector that is normal to its corresponding geometry in a given integration method
+     * @param IntegrationPointIndex index of integration point which jacobians has to be calculated in it.
+     * @param rIntegrationPointsArray The integration method which jacobians has to be calculated in its integration points.
+     * @return The normal in the given integration method
+     */
+    virtual array_1d<double, 3> Normal(
```

Comment:
Why we need to have this new interface. We can clearly use the below one passing the integration point as local coordinates as you did in line 1293. So what would be the benefit?